css/Properties/Properties.scss

```diff
@@ -199,3 +199,8 @@ $property-ext-padding-right: 8px;
 		margin-right: 0;
 	}
 }
+
+.property + .property.property-x-phonetic-first-name,
+.property + .property.property-x-phonetic-last-name {
+	margin-top: -40px;
+}
```

Comment:
![Capture d’écran_2020-08-11_09-04-10](https://user-images.githubusercontent.com/14975046/89867566-a6104580-dbb1-11ea-979d-527cadfa76ad.png)

Please remove those lines :)

Reply:
I wanted the Phonetic Name field to display as a concatenation with the previous N fields rather than as a separate.
But, It seems to have a layout that I didn't intend.
How about below?
Instead of concatenating with the N field, I just combined Phonetic First Name and Phonetic Last Name.

```suggestion
.property.property-x-phonetic-first-name + .property.property-x-phonetic-last-name {
	margin-top: -40px;
}
```

Reply:
Thi si snot possible, the order might be different and the structure might split in the middle.
You have no choice but to make them standalone elements :)

Reply:
OK. I understand.
Unfortunately, I gave up on the concatenating.
I remove this changes.